mmengine/visualization/vis_backend.py

```diff
@@ -411,6 +411,8 @@ def add_image(self,
             step (int): Useless parameter. Wandb does not
                 need this parameter. Default to 0.
         """
+        import wandb
+        image = wandb.Image(image)
```

Comment:
should we use `self._wandb.Image(image)`?

Reply:
`wandb.Image` also works, but `self._wandb.Image(image)` looks better. Thank you.